frontend/pages/admin/IntegrationsPage/cards/Mdm/components/EditTeamModal.tsx

```diff
@@ -1,29 +1,83 @@
-import React from "react";
+import React, { useState, useContext, FormEvent } from "react";
+
+import { AppContext } from "context/app";
+import configAPI from "services/entities/config";
 
 import Modal from "components/Modal";
 import Button from "components/buttons/Button";
 
+// @ts-ignore
+import Dropdown from "components/forms/fields/Dropdown";
+
 interface IEditTeamModal {
   onCancel: () => void;
-  onEdit: () => void;
+  currentDefaultTeamName: string | undefined;
```

Comment:
small nitpick but this is a verbose prop name. I think something like `defaultTeam` is sufficient. what do you think?

Reply:
`currentDefaultTeamName` refers to the current state of the server, in contrast to `defaultTeamName`, which is a state variable within the modal (set by default to `currentDefaultTeamName`)

In general I used the suffix 'Name' here because this is specifically dealing with the team name, a string, whereas a 'team' elsewhere generally refers to an object with lots more data than just its name. This stems from the fact that the API this modal hits is set up to take the team name as a string.

So that was my thinking naming-wise. Also, I used to use more concise variable names, but was then advised to use longer, very descriptive names, both by a mentor and the book "Clean Code." I see the appeal of both approaches, so if you still think more concise names would be better I can lean towards that approach here and in the future.